src/cloud/onboarding/onboarding-tasks.md

```diff
@@ -7,80 +7,120 @@ functional_areas:
+To quickly onboard your project, so you can begin developing your site for live deployment, you must complete required setup and onboarding tasks. Typically, the License Owner begins the process by securing Admin access and creating Technical Admin users that can help with setup, customization, and development work.
 
-## Project portal
+{:.bs-callout-info}
+If you need to change the License owner on your account, you must submit a Magento Support ticket.
 
-Once you have an account, you can use the Magento Project Portal to manage the project for your {{site.data.var.ece}} store. The portal provides a Getting Started guide and an interactive workflow that helps project administrators, business users, and developers with the tasks required to develop, build, test, and launch your site.
+## Sign up for a Cloud account {#cloud-first-acct}
 
-**Open the Magento Project Portal:**
+If you do not have a {{site.data.var.ece}} account, contact [Magento Sales][]. We will create your account and send you a welcome email that provides instructions to access the project.
 
-1. Log in to your Magento account.
+The person who signs up for a {{site.data.var.ece}} account is referred to as the _License Owner_.
 
-1. On the My Account page, click the Magento tab to see the projects in your account.
+## Your welcome email {#email}
 
-1. **Click View Project Page** in the [Projects section](https://cloud.magento.com/cloud/project/subscriptions/).
+Magento sends a welcome email to the License Owner using the address that was provided during the sign up process. The email contains a link to access your {{site.data.var.ece}} project so you can complete initial project setup.
 
-## Sign up for a {{site.data.var.ece}} account {#cloud-first-acct}
+You can access your project by logging in to [your {{site.data.var.ece}} account][].
 
-Don't have a {{site.data.var.ece}} account yet? Contact [Magento Sales](https://magento.com/explore/contact-sales). We will create your account and send you a welcome email that provides instructions to access the project.
+## Cloud Project portal (Onboarding UI)
 
-The person who signs up for a {{site.data.var.ece}} account is referred to as the _License Owner_.
+Use the {{site.data.var.ece}} Project Portal (Onboarding UI) to help project administrators, business users, and developers with the tasks required to develop, build, test, and launch your site.
+
+-  Access your Cloud environments, site, and Magento Admin
+-  Add a Technical Admin, a super user to help manage your project and branches
+-  Access your project environments, including a link to the Project Web Interface
+-  Provide your developer with a getting started workflow to set up their local development environments and begin developing
+-  Complete a quick user acceptance test (UAT) checklist with links to further tests
+
+**Open the Magento Project Portal:**
```

Comment:
This doesn't fit the [pattern for introducing a procedure](https://wiki.corp.magento.com/display/MDOC/Style+guide+-+conventions#Styleguide-conventions-Introduceaprocedure ). If you choose not to use procedure function, but a heading, then remove the colon.